packages/oui-collapsible/src/collapsible.controller.js

```diff
@@ -38,5 +38,7 @@ export default class {
 
     toggle () {
         this.expanded = !this.expanded;
+        const expanded = this.expanded;
+        this.onToggle({ expanded });
```

Comment:
```suggestion
        this.onToggle({ expanded: this.expanded });
```

I think we could go with this suggestion. It's shorter and still readable ;)